File: SDKs/circuitsandbox.net/2.9.233/python/src/sdk/models/operations/getfavoriteconversations.py
from dataclasses import dataclass, field
from typing import Optional
from sdk.models import shared


@dataclass
class GetFavoriteConversationsSecurity:
    oauth: shared.SchemeOauth = field(metadata={'security': { 'scheme': True, 'type': 'oauth2' }})
    

@dataclass
class GetFavoriteConversationsRequest:
    security: GetFavoriteConversationsSecurity = field()
    

@dataclass
class GetFavoriteConversationsResponse:
    content_type: str = field()
    status_code: int = field()
    body: Optional[bytes] = field(default=None)
    

File: SDKs/circuitsandbox.net/2.9.233/typescript/src/sdk/models/operations/getfavoriteconversations.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import * as shared from "../shared";



export class GetFavoriteConversationsSecurity extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "security, scheme=true;type=oauth2" })
  oauth: shared.SchemeOauth;
}


export class GetFavoriteConversationsRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  security: GetFavoriteConversationsSecurity;
}


export class GetFavoriteConversationsResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  body?: Uint8Array;

  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}
